ckanext/data_qld_theme/templates/package/read_base.html

```diff
@@ -0,0 +1,10 @@
+{% ckan_extends %}
+
+{% block content_primary_nav %}
+    {{ h.build_nav_icon('dataset_read', _('Dataset'), id=pkg.name) }}
+    {% if h.ytp_comments_show_comments_tab_page() %}   
```

Comment:
Where is this function defined? I can't find it in ckanext-data-qld or ckanext-data-qld-theme

Reply:
is in the comments plugin, my suggestion is that this injects/updates it via the plugin instead of inside the theme which now makes the theme dependant on comments to complete integration testing